packages/node-http-handler/src/node-http-handler.ts

```diff
@@ -151,7 +156,16 @@ export class NodeHttpHandler implements HttpHandler {
         };
       }
 
+      if (typeof nodeHttpsOptions.agent !== "boolean") {
+        // @ts-ignore
+        const keepAliveOptions: SocketKeepAliveOptions = nodeHttpsOptions.agent;
+        setSocketKeepAlive(req, {
+          keepAlive: keepAliveOptions.keepAlive,
+          keepAliveMsecs: keepAliveOptions.keepAliveMsecs
+        });
+      }
```

Comment:
The variable stored here is httpAgent
```suggestion
        const httpAgent = nodeHttpsOptions.agent;
        setSocketKeepAlive(req, {
          keepAlive: httpAgent.keepAlive,
          keepAliveMsecs: httpAgent.keepAliveMsecs
        });
      }
```